src/AddressRegistry/StreetName/StreetNameAddress_State.cs

```diff
@@ -353,5 +353,19 @@ private void When(AddressWasRemovedBecauseStreetNameWasRemoved @event)
 
             _lastEvent = @event;
         }
+
+        public void Readdress(ReaddressAddressData readdressItem, StreetName.StreetNameWasReaddressed @event)
```

Comment:
Can we make this method internal instead of public? I know we have other public methods, but this is about setting state.